turbopack/crates/turbopack-core/src/resolve/mod.rs

```diff
@@ -3079,76 +3081,70 @@ pub enum ModulePart {
     Facade,
 }
 
-#[turbo_tasks::value_impl]
 impl ModulePart {
-    #[turbo_tasks::function]
-    pub fn evaluation() -> Vc<Self> {
-        ModulePart::Evaluation.cell()
+    pub fn evaluation() -> Self {
+        ModulePart::Evaluation
     }
-    #[turbo_tasks::function]
-    pub fn export(export: RcStr) -> Vc<Self> {
-        ModulePart::Export(ResolvedVc::cell(export)).cell()
+
+    pub fn export(export: RcStr) -> Self {
+        ModulePart::Export(ResolvedVc::cell(export))
     }
```

Comment:
This has some behavior change since the export name is no longer a singleton, but we have many cells with the same export name.

This need to stay a turbo_tasks::function, so does all the others that cell something

Reply:
Or alternatively make `ResolvedVc<RcStr>` -> `RcStr` in the fields of ModulePart